examples/platforms/nrf528xx/src/radio.c

```diff
@@ -687,6 +722,7 @@ otError otPlatRadioSetTransmitPower(otInstance *aInstance, int8_t aPower)
 
     return OT_ERROR_NONE;
 }
+#endif
```

Comment:
```suggestion
#endif // !OPENTHREAD_CONFIG_ENABLE_PLATFORM_FEM_CUSTOM_SOURCE
```
More than 20 lines between the `#if`.

Reply:
Updated